chained/common.go

```diff
@@ -78,6 +78,14 @@ func _settingBool(settings map[string]string, name string) bool {
 	return val
 }
 
+func _settingBytes(settings map[string]string, name string) []byte {
+	_val := _setting(settings, name)
+	if _val == "" {
+		return nil
+	}
+	return []byte(_val)
+}
```

Comment:
We won't be able to pass raw bytes in this config.  The config is sent and parsed a YAML - a text-based format.  Including raw UTF-encoded bytes will potentially (likely) screw up the encoding of the YAML document.

Similar to the flag passed to [http-proxy](https://github.com/getlantern/http-proxy/pull/622/files#diff-dddc5dd1f92ee518b251cc43ab54489f6e4a3132e6e857baecabb3d271fe8ab2R190), this will need to be encoded in a text-friendly format.  Base64 would work well here too.

Reply:
Oh, I see that you're treating the WASM as a base64-encoded string in water_impl.go.  So I guess you figured this out already 🙂 

I think this is dead code then, right?  Looks like it's no longer used, so we should just get rid of it.